sa/sa_test.go

```diff
@@ -2574,3 +2578,139 @@ func TestIncidentsForSerial(t *testing.T) {
 	test.AssertNotError(t, err, "Failed to retrieve incidents for serial")
 	test.Assert(t, len(incidentsForSerial) == 1, "No active incidents returned")
 }
+
+type inmemSA struct {
+	sapb.StorageAuthorityClient
+	Impl *SQLStorageAuthority
+}
```

Comment:
Why have this separate from the `inmem/sa` package?

Reply:
Unfortunately the `inmem/sa` package imports `sa`. So, importing it here in the `sa` would lead to a cyclical import.